test/services/listCourtCases.integration.test.ts

```diff
@@ -1570,43 +1568,37 @@ describe("listCourtCases", () => {
       visibleForces: [orgCode],
       visibleCourts: [],
       groups: [],
-      hasAccessToExceptions: false,
-      hasAccessToTriggers: false
+      hasAccessTo: userAccess({ groups: [] })
     } as Partial<User> as User
     const triggerHandlerUser = {
       visibleForces: [orgCode],
       visibleCourts: [],
       groups: [UserGroup.TriggerHandler],
-      hasAccessToExceptions: false,
-      hasAccessToTriggers: true
+      hasAccessTo: userAccess({ groups: [UserGroup.TriggerHandler] })
```

Comment:
Could possibly refactor these tests to make use of the defined helpers, for instance:
`hasAccessTo: generalHandlerHasAccessTo`

Reply:
I've switched this over to use the helpers